cpu/kinetis/Makefile.features

```diff
@@ -1,5 +1,18 @@
 FEATURES_PROVIDED += periph_cpuid
-FEATURES_PROVIDED += periph_hwrng
+
+# HACK Do not define 'hwrng' if the board does not supports it
+# I prefer a whitelist on CPU_MODEL but the information is not available yet
+# HACK the board/cpu_model currently uses the wrong hwrng register
```

Comment:
If I understand this correctly, the wrong hwrng register is used for frdm-k64f ? Maybe make this more explicit that this HACK applies to frdm-kw64f.

Reply:
I forgot to re-precise this when moving it to the common directory.

Reply:
While re-reading I am not sure if it is the `frdm-k64f` or the `CPU_MODEL` completely. That is why I wrote `board/cpu_model`. I think it may be more cpu_model but hard to know without debugging why.

I can change to `Hack, either the board or the cpu_model currently uses the wrong hwrng` maybe ?

Reply:
Maybe start the next sentence on the same line:
`HACK the board/cpu_model currently uses the wrong hwrng register. Remove next line when fixed` (you can split it if it's too long)

Reply:
I can just remove the "remove next line when fixed" as it is implicit and still precise the `frdm` as it is currently the only one I know as you said in the first comment.
`The HWRNG uses the wrong hwrng register for the frdm-k64f board/cpu_model`